packages/core/src/html/NodeProcessor.ts

```diff
@@ -265,13 +264,26 @@ export class NodeProcessor {
          */
         if (!_.has(node.attribs, 'v-pre')) { node.attribs['v-pre'] = ''; }
         break;
+      case 'pic':
+      case 'annotate':
+        if (_.has(node.attribs, 'lazy')
+            && !(_.has(node.attribs, 'width') || _.has(node.attribs, 'height'))) {
+          const filePath = context.callStack.length > 0 ? context.callStack[context.callStack.length - 1]
+            : context.cwf;
+          logger.warn(
+            `${filePath} --- `
+              + 'Both width and height are not specified when using lazy loading in the file and'
+              + ' it might cause shifting in page layouts. '
+              + 'To ensure proper functioning of lazy loading, please specify either one or both.\n',
+          );
+        }
+        break;
       default:
         break;
       }
     } catch (error) {
       logger.error(error);
     }
-
```

Comment:
I think the newlines were intentional for readability! Perhaps reverting them could prevent any unnecessary change!

Reply:
Ok, will make the changes!